### PR TITLE
pj-106/107 add notification messages, some update views code, some im…

### PR DESCRIPTION
**Pull request by NykoEugen**
…proves

### CODE REVIEW

**Comment by NykoEugen** [comment]
tests fix later

File: static/js/category.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+document.addEventListener("DOMContentLoaded", function () {
```
Comment (acman) [inline static/js/category.js:1]: I am not sure I understand what is purpose for it? Could be added some comments to clarify the code?

File: static/js/post.js [hunk 2]
```diff
@@ -140,4 +140,11 @@ document.addEventListener("DOMContentLoaded", function () {
       cancelButton.addEventListener("click", cancelButton.clickHandler);
     });
   });
+  // Notification about actions was completed successfully
+  let messageBlock = document.getElementById("notification-messages");
+  console.log(messageBlock)
```
Comment (acman) [inline static/js/post.js:145]: console logs should be removed

**Review by acman — CHANGES_REQUESTED** [review]
Looks good! Needs some clarification.

File: static/js/category.js (hunk 1 above)
Comment (NykoEugen) [reply static/js/category.js:1]: maybe it should be add in main JS. This is notification about some actions(create and limitation)

**Comment by acman** [comment]
Don't see any comments in code, try to describe code and add some examples to be clear for other developers :) especially in JS :)

**Comment by acman** [comment]
and there are some conflicts

**Review by acman — APPROVED** [review]
👍

**Pull request merged**